src/lib/apollo/initApollo.js

```diff
@@ -66,10 +83,9 @@ const create = (initialState, options) => {
     }
   }));
 
-  const httpLink = new HttpLink({ uri: `${graphqlUrl}`, credentials: "same-origin" });
+  const httpLink = new HttpLink({ uri: graphqlUrl, credentials: "same-origin" });
 
   return new ApolloClient({
-    connectToDevTools: process.browser,
```

Comment:
Apollo dev tools wasn't working for me anymore until I removed this. It automatically enables only for non-production, so it should be fine.